spec/models/ride_spec.rb

```diff
@@ -3,5 +3,8 @@
 RSpec.describe Ride, type: :model do
   describe "relationships" do
     it { should belong_to(:amusement_park) }
+    it { should have_many(:mechanic_rides) }
+    it { should have_many(:mechanics).through :mechanic_rides }
+
```

Comment:
Nice work with testing overall! When you encapsulate the logic from US 3 in a model method, though, it'd be good to add a model method test for that functionality